src/Components/Topbar/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+export interface TopBarProps {
+  /** 왼쪽방향 아이콘과 제목 사이의 gap */
+  gap?: number;
+
+  /** Topbar Title content */
+  children?: React.ReactNode;
+}
+
+export const TopBar = ({ gap = 12, children }: TopBarProps) => {
+  const navigate = useNavigate();
+  return (
+    <TopbarWrapper gap={gap}>
+      <MoveBack onClick={() => navigate(-1)}>
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+          <path
+            d="M17.4396 24C17.2348 24 17.03 23.922 16.874 23.7656L6.21997 13.112C5.60677 12.4988 5.60677 11.5012 6.21997 10.8884L16.874 0.234422C17.1864 -0.0779779 17.6928 -0.0779779 18.0052 0.234422C18.3176 0.546822 18.3176 1.05322 18.0052 1.36562L7.37117 12L18.0056 22.6344C18.318 22.9468 18.318 23.4532 18.0056 23.7656C17.8492 23.922 17.6448 24 17.44 24H17.4396Z"
+            fill="black"
+            fill-opacity="0.65"
+          />
+        </svg>
+      </MoveBack>
+      <TopBarTitle>{children}</TopBarTitle>
+    </TopbarWrapper>
+  );
+};
+
+const TopbarWrapper = styled.div<{ gap?: number }>`
+  display: flex;
+  flex: 1 0 0;
+  max-width: 1224px;
+  width: 100%;
+  align-items: center;
+  gap: ${({ gap }) => `${gap}px`};
+`;
+
+const MoveBack = styled.div`
```

Comment:
```suggestion
const MoveBack = styled.button`
```

혹시 뒤로가기를 div로 구현하신 이유가 있을까요?

`button`을 사용하면 스크린 리더 등 보조 기기가 해당 아이콘을 버튼으로 인식하여 사용자가 클릭 가능한 버튼임을 인지할 수 있어서 접근성 면에서도 좋을 것 같습니다!

https://techblog.woowahan.com/15541/

Reply:
구현하면서도 찝찝함이 남아있었는데, 접근성을 고려하면 button으로 구현하는 게 맞을 것 같네요!! 감사합니다🙏